src/index.html

```diff
@@ -9,10 +10,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link references a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled to CSS first. Ensure that `styles/main.scss` is compiled to a CSS file and update the `href` attribute to point to the compiled CSS file.